packages/trimerge-sync/src/TrimergeClient_Remote.test.ts

```diff
@@ -14,21 +14,20 @@ import { SyncStatus } from './types';
 import { timeout } from './lib/Timeout';
 import { resetAll } from './testLib/MemoryBroadcastChannel';
 
-type TestCommitMetadata = string;
+type TestEditMetadata = string;
 type TestSavedDoc = any;
 type TestDoc = any;
 type TestPresence = any;
 
-const differ: Differ<TestSavedDoc, TestDoc, TestCommitMetadata, TestPresence> =
-  {
-    migrate,
-    diff,
-    patch,
-    computeRef,
-    mergeAllBranches,
-  };
+const differ: Differ<TestSavedDoc, TestDoc, TestEditMetadata, TestPresence> = {
+  migrate,
+  diff,
+  patch,
+  computeRef,
+  mergeAllBranches,
+};
```

Comment:
should we add a defaultDiffer to merge utils?

Reply:
we seem to have this in a few places

Reply:
I'd be fine with that